app/js/rest/encounterRest.js

```diff
@@ -0,0 +1,35 @@
+
+import axiosInstance from '../config';
+
+const api = {
```

Comment:
Ideally, we'd use the encounter REST from react-components, but you were saying there was a problem with the format that the encounterRest in react-components returned? Or was it something else?

Reply:
@mogoodrich yeah, it was returning the location as null

Reply:
Strange, it looks like the default REST rep for Encounter in react-components includes "location"... (and it should).